src/bridge/IrcBridge.ts

```diff
@@ -926,10 +926,21 @@ export class IrcBridge {
         ));
     }
 
+    /**
+     * Determines if a nick name already exists.
+     */
     public async checkNickExists(server: IrcServer, nick: string) {
         log.info("Querying for nick %s on %s", nick, server.domain);
         const client = await this.getBotClient(server);
-        return await client.whois(nick);
+        try {
```

Comment:
Hmm, any reason why this is on the `IrcBridge` monolith and not on `BridgedClient` which the user is using to change their nick?

Reply:
The function wasn't used anywhere.
I'll happily move it to `BridgedClient`.